engine/dlib/src/test/test_configfile.cpp

```diff
@@ -22,9 +22,8 @@
 #define JC_TEST_IMPLEMENTATION
 #include <jc_test/jc_test.h>
 
-// We're currently disabling some tests on win32 host
-#if defined(_WIN32) || defined(DM_PLATFORM_VENDOR) || defined(__EMSCRIPTEN__)
-#define SUPPORT_HTTP
+#if (defined(GITHUB_CI) && defined(__MACH__)) || defined(_WIN32) || defined(DM_PLATFORM_VENDOR) || defined(__EMSCRIPTEN__)
+#define DM_DISABLE_HTTPCLIENT_TESTS
```

Comment:
old `SUPPORT_HTTP` was unclear (we were checking for `ifndef` but it's called like if it should be positive check), so I renamed it to use the same name as in [engine/dlib/src/test/test_httpclient.cpp](https://github.com/defold/defold/pull/11563/files#diff-6a469696b272eb3789c4bd6026c4a9bc112b8dc0edff2c228d394e016ec2fa51)